data/mods/Xedra_Evolved/spells/XAEA.json

```diff
@@ -210,7 +210,7 @@
     "name": [ "The Knowledge" ],
     "desc": [ "You know for sure - right now you are the smartest human on this side of the planet." ],
     "apply_message": "With some effort you can outsmart reality.",
-    "remove_message": "You feel yourself much less smarter than before.",
+    "remove_message": "You less intelligent than before.",
```

Comment:
```suggestion
    "remove_message": "You feel yourself much less smarter than before.",
```

Reply:
I think this should be 'less smart' (as usual in language, this is a matter of opinion). (If you know differentiation, 'smarter' as a proposition sounds like "f'(x)>0", and numerically "degree of being smarter = f'(x)", so 'less smarter' would be like "f''(x)<0"  or "degree of being less smarter = f''(x)" - though, in this case discrete differentials might be more appropriate.)

Apparantly this has as name "double comparatives" <https://en.wikipedia.org/wiki/Double_comparative> (I don't buy 'Shakespeare said it so it's fine' or the hypocritical 'brr prescriptivism bad, _and_ I'm prescribing to you to consider my opinion correct and yours wrong' excuse. There is something to be said about not-prescriptivism, but so often it is turned into a prescriptivism of its own.)